test/scripts/build.actions.test.js

```diff
@@ -97,9 +97,18 @@ describe('build by zipping js action folder', () => {
     expect(utils.zip).toHaveBeenCalledWith(r('/actions/action-zip'), r('/dist/actions/action-zip.zip'))
   })
 
-  test('should fail if no package.json', async () => {
+  test('should not fail if no package.json if there is an index.js', async () => {
```

Comment:
should we also add `should fail if no package.json and no index.js` ? or is it covered already?

Reply:
The next test is literally `test('should fail if no package.json and no index.js', async () => {`